extensions/replication/queuePopulator/LogReader.js

```diff
@@ -246,23 +246,45 @@ class LogReader {
     }
 
     _logEntryToQueueEntry(record, entry) {
+        // FIXME: copied from S3, move those values to arsenal from
+        // S3/constants.js and use them
+        const excluded = {
+            usersBucket: 'users..bucket',
+            oldUsersBucket: 'namespaceusersbucket',
+            mpuBucketPrefix: 'mpuShadowBucket',
+        };
+
         if (entry.type === 'put' &&
-            entry.key !== undefined && entry.value !== undefined &&
-            !isMasterKey(entry.key)) {
-            const value = JSON.parse(entry.value);
-            if (value.replicationInfo &&
-                value.replicationInfo.status === 'PENDING') {
-                this.log.trace('queueing entry', { entry });
-                const queueEntry = {
-                    type: entry.type,
-                    bucket: record.db,
-                    key: entry.key,
-                    value: entry.value,
-                };
-                return {
-                    key: encodeURIComponent(entry.key),
-                    message: JSON.stringify(queueEntry),
-                };
+            entry.key !== undefined && entry.value !== undefined) {
+            const isVersionedKey = !isMasterKey(entry.key);
+            const metaStoreOp = record.db === '__metastore';
+            if (isVersionedKey || metaStoreOp) {
```

Comment:
Can this be removed and can the inner `if` statement be used by itself or is this a quick check to avoid use of JSON.parse below?